src/IssueViz/Selection/IAnalysisIssueSelectionService.cs

```diff
@@ -32,7 +32,7 @@ internal interface IAnalysisIssueSelectionService : IDisposable
         IAnalysisIssueLocationVisualization SelectedLocation { get; set; }
     }
 
-    internal enum SelectionChangeLevel
+    public enum SelectionChangeLevel
```

Comment:
Has to be public, or else I can't use it in tests' DataRow

Reply:
Ok. FYI I passed it as a string in my data tests and used `Enum.Parse`.